### ISSUE TITLE
Gesture lock is missing instructions on how to unlock

### ISSUE THREAD

**Issue by Zinurist** [jellyfin/Swiftfin#1236]
I don't know how, but I enabled gesture lock by accident. As expected, it rendered the player unusable.

I tried various gestures to unlock it again, but failed to do so, so I had to restart the app. Stuff like double taps, long pressing, on the gesture lock message or elsewhere, didn't work. I'm also failing to enable gesture lock again.

My suggestion would be to add instructions on how to unlock it in the gesture lock pop-up message.

**Comment by deprecastor** [comment]
Same for me. Locked the second time and now seeking help to unlock. But cannot find it.

**Comment by LePips** [comment]
Yes this is a bad interaction and will be fixed in #1203. You can disable the gesture lock by tapping the toast view that appears while gesture lock is on.

**Comment by te-simps** [comment]
I'm having the same issue. What do you mean by "toast viee"?  It would be good to have some clear instructions on how to re-enable gestures.

**Comment by soudasuwa** [comment]
Lock/unlock gestures: long press anywhere on the screen for like 5 seconds.

**Comment by rvegele** [comment]
Just had this. Gesture lock enabled somehow and no instruction how to unlock. This should be added as a part of lock message. “Press and hold on screen to unlock”.

**Comment by JPKribs** [comment]
This should be resolved as part of: https://github.com/jellyfin/Swiftfin/pull/2195
